autoprecompiles/src/lib.rs

```diff
@@ -120,6 +123,17 @@ impl<T: Display> Display for SymbolicMachine<T> {
     }
 }
 
+impl<T: Clone + Ord + std::fmt::Display> SymbolicMachine<T> {
+    pub fn degree(&self) -> usize {
```

Comment:
Just curious: we were talking about having two degrees, one for constraints and one for bus interactions. How does  this method relate to that? Is the bus interaction one this one minus one?

Reply:
Could be that yea, but I guess we can also make this 2 functions at that point